lldb/test/API/functionalities/disassembler-variables/TestRichDisassembler.py

```diff
@@ -18,16 +18,34 @@ def _compile_object(self, src_name, func_cflags="-g -gdwarf-5 -O2 -fno-inline"):
         self.assertTrue(os.path.exists(obj), f"missing object: {obj}")
         return obj
 
+    def _compile_or_assemble_object(
+        self,
+        src_name,
+        cflags="-g -gdwarf-5 -O2 -fno-inline -fno-omit-frame-pointer",
```

Comment:
If the input is assembler, the. -O2 (and the -f flags) should have no effect, so you can remove them